server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/DeleteAction.java

```diff
@@ -53,19 +57,21 @@ public void define(NewController controller) {
       .setPost(true)
       .setHandler(this);
 
-    QProfileRef.defineParams(action, languages);
+    qProfileWsSupport.createOrganizationParam(action)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2209)